README.md

```diff
@@ -37,10 +37,10 @@ or you can manually download the jar from [Bintray](https://bintray.com/linkedin
 
 dex-test-parser provides a single public method that you can call from Java to get all test method names.
 ```java
-
-List<String> testMethodNames = DexParser.findTestNames(apkPath);
-
+List<String> customAnnotations = listOf("");
```

Comment:
This is a mixture of java and Kotlin code. We can update the language to be java